### PR TITLE
Prmdr 379 2

### PR DESCRIPTION
**Pull request by SRAlexander**

### CODE REVIEW

File: lambdas/handlers/authoriser_handler.py
Comment (danielleandersonNHS) [inline lambdas/handlers/authoriser_handler.py:1]: Need to remember to turn back on the routing for PCSE

File: app/src/pages/patientSearchPage/PatientSearchPage.story.tsx
Comment (danielleandersonNHS) [inline app/src/pages/patientSearchPage/PatientSearchPage.story.tsx:1]: Redundant import for USER_ROLE

**Review by danielleandersonNHS — CHANGES_REQUESTED** [review]
Few redundant imports but overall code changes are fine
Additional comment to ensure that PSCE routing is addressed following merge

File: app/src/pages/patientResultPage/PatientResultPage.story.tsx
[the remark this answers is not in the document]
Comment (thisusernameisnowtaken) [reply app/src/pages/patientResultPage/PatientResultPage.story.tsx:1]: Import removed

File: lambdas/handlers/authoriser_handler.py
Comment (thisusernameisnowtaken) [reply lambdas/handlers/authoriser_handler.py:1]: We'll revisit this when we merge Rio's branch

**Pull request merged**